<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QWidget" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>1271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Graphic Scene</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>470</y>
     <width>1011</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>0</y>
     <width>20</width>
     <height>701</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>601</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upload the obj file to add a model on the scene. Choose it from
your local storage or search the web catalogue to download.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fullscreen_pb">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>420</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: green;
border-width: 10px</string>
   </property>
   <property name="text">
    <string>Fullscreen mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_world_pb">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>70</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Scene objects</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pb">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>600</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>-1</y>
     <width>20</width>
     <height>481</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>World Settings</string>
   </property>
  </widget>
  <widget class="QListWidget" name="scene_lw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>211</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save_pb">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>250</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>500</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Press ESC to exit
fullscreen mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload_file_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Upload an obj file</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>500</y>
     <width>391</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select the object you want to change. You
may switch between skyboxes by pressing T.
- scroll to zoom
- AWSD to move
- move to rotate
-Del to delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="init_skybox_pb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>610</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Init skybox</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>330</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Save your progress to
continue working later</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>150</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upload a whole
world from a file</string>
   </property>
  </widget>
  <widget class="OGLWidget" name="canvas_widget" native="true">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>47</y>
     <width>775</width>
     <height>432</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QWidget</extends>
   <header>oglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
